observatory-reports/requirements.txt

```diff
@@ -0,0 +1,9 @@
+observatory-platform
+matplotlib==3.1.*
+pandas==0.25.*
+seaborn==0.10.*
```

Comment:
Should this be 0.11.* ?

Reply:
Thanks Cameron, yeah you're right, I've changed it to that version now.